donate/payments/constants.py

```diff
@@ -528,3 +528,1247 @@
   'XOF',
   'XPF'
 ]
+
+
+# List of countries and their respective post code formats if applicable.
+# Used in post-code-validation.js to check whether or not we display a post code form item to the user.
+# Source of data can be found here: https://gist.github.com/lkopocinski/bd4494588458f5a8cc8ffbd12a4deefd
+# NOTE: This list is also found in donate/constants.py, any updates to this list should be made there/vice-versa.
+COUNTRY_POST_CODES = [
```

Comment:
Both Python and JavaScript can load in .json data, so if we need this list to be synced between the two, let's just use the JSON from the gist, and have both load that in, instead of having two files that can go out of sync.

(but remember that json parsing usually needs a try/catch or try/except)